### DISCUSSION TITLE
Can't find neither branches nor tags labelling versions other than 6.07 and 6.07.1

### DISCUSSION THREAD

**Discussion by Syuf1514** [NOAA-EMC/WW3#1120] [General]
I want to install latest WW3 model from github source code, but i found only  6.07 and 6.07.1 tags and no other version labelling there. However some report issues about versions 7.+ of WW3, where do they get it?

**Comment by aliabdolali** [comment]
you should try the top of develop which is v7.14 and you can navigate backward to older versions.

**Comment by ukmo-ccbunney** [comment]
Hello @Syuf1514 
There are no newer tagged releases at the moment. 
However, you can access the most up-to-date development by checking out the [`develop`](https://github.com/NOAA-EMC/WW3/tree/develop)  branch of the repository.
This is actually the default branch, so if you clone the repository you will get the head of develop anyhow (which is currently at version 7.14 I believe).

**Comment by Syuf1514** [comment]
Thanks. It would be nice though to have tags for all the versions, so i can git clone --branch "v7.x.x" --depth 1 desirable ww3 version (which may not be the latest) without downloading excessive git history. Thanks again for the quick response.
